src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -39,24 +41,19 @@ describe('convertInclude', () => {
 
   it('should fetch and convert an include block to an idt1 block', async () => {
     fetch.mockResponse(() => Promise.resolve(idt1Markup));
-    const input = {
-      required: false,
-      tile: 'A quiz!',
-      href: '/indepthtoolkit/quizzes/123-456',
-      platform: 'highweb',
-      type: 'include',
-    };
+    const input = blocks[0];
```

Comment:
I think the removal of the inline inputs make the tests more difficult to read though I appreciate you now need to pass in the page data (something that again make me uncomfortable with the index approach).

If we stick with this approach, could we maybe extract the blocks at the top of the test file into consts that describe the characterstics of each block e.g.
`const [ idt1Block, idt2Block, unsupportedIncludeBlock, ...] = blocks;`
Just makes it easier to understand, at a glance what the input block structure is expected to be.
```suggestion
    const input = idt1Block;
```